src/foundry/foundry.js/pixi/containers/placeableObjects/tile.d.ts

```diff
@@ -100,7 +100,9 @@ declare global {
     protected _onDragLeftMove(event: PIXI.InteractionEvent): void;
 
     /** @override */
-    protected _onDragLeftDrop(event: PIXI.InteractionEvent): Promise<false | this | BaseTile[]>;
+    protected _onDragLeftDrop(
+      event: PIXI.InteractionEvent
+    ): Promise<false | this | InstanceType<ConfiguredDocumentClass<typeof TileDocument>>[]>;
```

Comment:
once again, might make sense to just type the return type loosely as `Promise<unknown>` or similar.